client/src/containers/UpdatePassword/index.tsx

```diff
@@ -0,0 +1 @@
+export { default } from "./UpdatePassword";
```

Comment:
For what we need it?

Reply:
Just compare importing this component in App.tsx file

    import SignUp from "./containers/SignUp/SignUp";
    import ResetPassword from "./containers/ResetPassword";